mailpoet/tsconfig.json

```diff
@@ -25,6 +25,7 @@
     "allowJs": true,                        /* Allow javascript files to be compiled. */
     "checkJs": false,                        /* Report errors in .js files. */
     "resolveJsonModule": true,              /* Allow importing json files */
+    "skipLibCheck": true,                   /* Skip type checking all .d.ts files. See more: https://www.typescriptlang.org/tsconfig#skipLibCheck */
```

Comment:
This is semi-related, maybe it needed it's own separate commit, this is fixing the issue we sometimes experience where TS finds multiple definitions for React types since several of our dependencies have React or related packages as dependency.

Reply:
@samnajian I understand and agree with the change, it's just that we normally don't put such changes together with very different ones.

There's one more thing though - if we add `skipLibCheck`, we should then also remove [these lines](https://github.com/mailpoet/mailpoet/blob/a644c71e64b44196c4cb1a0d34d8b90fbf3b7a48/mailpoet/tasks/npm_post_install.sh#L24-L27).